measures/2019/measures-data.json

```diff
@@ -57511,15 +57515,15 @@
     "strata": [
       {
         "name": "counselalcohol",
-        "description": "Risk Assessment or Counseling for Alcohol Use"
+        "description": "Risk Assessment or Counseling for Alcohol Use,"
       },
       {
         "name": "counseltobacco",
-        "description": "Risk Assessment or Counseling for Tobacco Use"
+        "description": "Risk Assessment or Counseling for Tobacco Use,"
       },
       {
         "name": "counselsubstance",
-        "description": "Risk Assessment or Counseling for Other Substance Use"
+        "description": "Risk Assessment or Counseling for Other Substance Use, and"
```

Comment:
This also seems unusual @sarahnw

Reply:
I'd agree that this looks a bit strange.

Reply:
it's how it's formatted on the spreadsheet.